src/roman_datamodels/maker_utils/_common_meta.py

```diff
@@ -434,22 +435,47 @@ def mk_photometry_meta(**kwargs):
     return meta
 
 
-def mk_resample_meta(**kwargs):
+def mk_mosaic_meta(**kwargs):
     """
-    Create a dummy common metadata dictionary with valid values for attributes and add
-    the additional photometry AND resample metadata
+    Create a dummy metadata dictionary with valid values for mosaic attributes.
 
     Returns
     -------
-    dict (defined by the common-1.0.0 schema with additional photometry and resample metadata)
+    dict (defined by the wfi_mosaic-1.0.0 schema)
     """
 
-    meta = mk_photometry_meta(**kwargs)
+    meta = {}
+    meta["asn"] = mk_mosaic_associations(**kwargs.get("asn", {}))
+    meta["basic"] = mk_mosaic_basic(**kwargs.get("basic", {}))
+    meta["cal_step"] = mk_cal_step(**kwargs.get("cal_step", {}))
+    meta["coordinates"] = mk_coordinates(**kwargs.get("coordinates", {}))
+    meta["individual_image_meta"] = mk_individual_image_meta(**kwargs.get("individual_image_meta", {}))
+    meta["photometry"] = mk_photometry(**kwargs.get("photometry", {}))
+    meta["program"] = mk_program(**kwargs.get("program", {}))
+    meta["ref_file"] = mk_ref_file(**kwargs.get("ref_file", {}))
     meta["resample"] = mk_resample(**kwargs.get("resample", {}))
+    meta["wcsinfo"] = mk_mosaic_wcsinfo(**kwargs.get("wcsinfo", {}))
 
     return meta
 
 
+def mk_mosaic_associations(**kwargs):
+    """
+    Create a dummy mosaic associations instance with valid values for
+    mosaic associations attributes. Utilized by the model maker utilities.
+
+    Returns
+    -------
+    roman_datamodels.stnode.MosaicAssociations
+    """
+
+    mosass = stnode.MosaicAssociations()
```

Comment:
Just an observation: we usually we abbreviate association as asn so mosasn instead of mosass.
You did get a good laugh out of me.